src/common/config_opts.h

```diff
@@ -1053,6 +1053,7 @@ OPTION(bluestore_debug_randomize_serial_transaction, OPT_INT, 0)
 OPTION(bluestore_inject_wal_apply_delay, OPT_FLOAT, 0)
 OPTION(bluestore_shard_finishers, OPT_BOOL, false)
 OPTION(bluestore_rtc, OPT_BOOL, false)
+OPTION(bluestore_no_finisher, OPT_BOOL, false)
```

Comment:
how about bluestore_rtc_sync_completions, and default to true?  (it'll only matter if bluestore_rtc is also true.)